build/split_man.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import sys
+
+if len(sys.argv) != 4:
+    print('usage:', sys.argv[0], '<input> <output> <lang>')
+    exit(1)
+
+input = sys.argv[1]
+output = sys.argv[2]
+lang = sys.argv[3].lower()
+
+if lang == 'c':
+    lang = b'c'

```

Comment:
I wonder if you could just convert `lang` to a byte string upon initialization by appending `.encode('ASCII')` (on line 11) and delete lines 14, 20 and 26.

Reply:
Sure, that would work too.